pkg/cloud/azure/services/virtualmachines/virtualmachines.go

```diff
@@ -34,6 +34,9 @@ import (
 	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/services/networkinterfaces"
 )
 
+// DefaultTimezone a default and only allowed value for windows machine timezone configuration
```

Comment:
Still not clear to me why are we setting this?
https://docs.microsoft.com/en-us/rest/api/compute/virtualmachines/createorupdate#windowsconfiguration
If empty Azure will just just UTC as for all services right?

The comment also does not seems accurate. There's multiple allowed values here:
https://www.terraform.io/docs/providers/azurerm/r/virtual_machine.html#timezone
https://jackstromberg.com/2017/01/list-of-time-zones-consumed-by-azure/
https://docs.microsoft.com/en-us/dotnet/api/system.timezoneinfo.getsystemtimezones?view=netcore-3.1